src/rgw/rgw_op.cc

```diff
@@ -4858,7 +4858,9 @@ void RGWPutLC::execute()
   string shard_id = s->bucket.tenant + ':' + s->bucket.name + ':' + s->bucket.bucket_id;  
   string oid; 
   get_lc_oid(s, oid);
-  pair<string, int> entry(shard_id, lc_uninitial);
+  cls_rgw_lc_entry entry;
```

Comment:
Not a big deal, but you could have a constructor for cls_rgw_lc_entry that takes the bucket.  And the default constructors should be setting status to lc_uninitial, if they aren't already.

Reply:
has been updated by following your comment.